### ISSUE TITLE
Underscore in field name

### ISSUE THREAD

**Issue by d4rk5eed** [thetron/mongoid-enum#12]
What is the reason for using underscore in fieldname?

~~~
> Meeting.new.attributes
=> {"_id"=>BSON::ObjectId('549d746563616c2168060000'), "_meeting_quality"=>:hd, "_status"=>:inactive}
~~~

Actually we get another attribute that differ one we created

**Comment by xpepermint** [comment]
Asking the same question! +1

**Comment by alexanderk23** [comment]
+1

**Comment by bhirsch42** [comment]
+1

**Comment by thetron** [comment]
To be honest - I can't 100% remember my motivations for doing this. If I had to guess - it was probably to indicate that the field shouldn't be directly access, except through the getters and setters - following on from a few other (similar) gems that do similar things.

@xpepermint What are your thoughts on strings vs. symbols? I felt like symbols would be a better choice here, given that the values are immutable, but curious to hear why you feel these should be strings.

**Comment by xpepermint** [comment]
@thetron Huh, I don't remember exactly why but I think I was looking at rails `enum` function which works on strings.

**Comment by nofxx** [comment]
There's 5+ forks only to remove the underscore. +1 for me hehe.

**Comment by thetron** [comment]
Thanks everyone again for your feedback here. I've merged in #12, which added a configuration option to customise the prefix. At some point in the future we might switch the default to be an empty string, but I wanted to make it easy for existing and future users to continue using the gem on the path to v1.0.0, and support for mongoid 5.

:cake:
